tools/sampling.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+
+# Load the ENEM dataset
+df = pd.read_csv('datasets/microdados_enem_2022/DADOS/MICRODADOS_ENEM_2022.csv', encoding='ISO-8859-1')  # Replace with the actual path to your ENEM dataset
```

Comment:
I had also to define a separator in my case ... , sep=";"). If it's working I am fine with it.